src/ui/ui_get_public_key.c

```diff
@@ -152,24 +149,25 @@ static void callback_match(bool match) {
 
 static void callback_export(bool accept) {
     if (accept) {
-        nbgl_useCaseAddressConfirmation((const char *) gpk_ctx.full_key, callback_match);
+        nbgl_useCaseAddressConfirmation((const char *)gpk_ctx.full_key,
```

Comment:
Should be replaced by `nbgl_useCaseAddressReview`?

Reply:
For hedera this not exactly the same use case as for other apps and we cannot really use address review.